script/server

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+bash fetch_and_build_all_images.sh
```

Comment:
If you add the shebang to the `fetch_and_build_all_images.sh` script and make it executable you don't have to call it here with bash. I would prefer to do it like this then:
```suggestion
./fetch_and_build_all_images.sh
```